frontend/src/components/place/PlaceMapMarker.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+    <mapbox
+        :access-token="getMapboxToken"
+        :map-options="{
+            style: getMapboxStyle,
+            center: currentCenter,
+            zoom: 11
+        }"
+        :scale-control="{
+            show: true,
+            position: 'top-left'
+        }"
+        @map-init="mapInitialize"
+        @map-load="mapLoaded"
+    />
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex';
+import Mapbox from 'mapbox-gl-vue';
+import markerManager from '@/services/map/markerManager';
+
+export default {
+    name: 'PlaceMapMarker',
+
+    data() {
+        return {
+            isMapLoaded: false,
+            map: {},
+            markerManager: null,
+        };
+    },
+
+    props: {
+        place: {
+            type: Object,
+            required: true
+        }
+    },
+
+    components: {
+        Mapbox
+    },
+
+    created() {
+        if (this.isMapLoaded) {
+            this.updateMap();
+            markerManager.fitMarkersOnMap();
+        }
+    },
+
+    methods: {
+        ...mapActions('place', ['mapInitialization']),
+
+        mapInitialize(map) {
+            if (this.mapInitialized) {
+                return;
+            }
+            this.map = map;
+            this.mapInitialization();
+        },
+
+        mapLoaded(map) {
+            this.markerManager = markerManager.getService(map);
+            this.isMapLoaded = true;
+        },
+
+        updateMap() {
+            if (this.isMapLoaded) {
+                this.markerManager.setMarkersFromPlacesAndFit(this.place);
```

Comment:
We can just call inside updateMap, so no need to have `markerManager: null,` in data {}
markerManager
  .getService(map)
  .setMarkersFromPlacesAndFit(this.place);